### PR TITLE
New aggreagated CA file models 

### PR DESCRIPTION
**Pull request by danielSanchezQ**
It contains a utils method to load data from a csv file

### CODE REVIEW

**Review by Zeegomo — COMMENTED** [review]
Besides the lints, LGTM

**Comment by danielSanchezQ** [comment]
@Zeegomo Lints are because this models have no use right now. They will be included in next PRs

File: src/community_advisors/models/de.rs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+use serde::Deserialize;
+
+use vit_servicing_station_lib::db::models::community_advisors_reviews::ReviewTag;
```
Comment (Zeegomo) [inline src/community_advisors/models/de.rs:3]: Got it, thanks. I think this is unused anyway though

**Comment by danielSanchezQ** [comment]
Im closing this. I'll need to adapt some stuff on the fly related to ca rewards. So this will be part of the other incoming PR

**Pull request closed without merging**